stock_picking_mgmt_weight/models/stock_picking.py

```diff
@@ -0,0 +1,48 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields, api, _
+from datetime import date, datetime
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    type_scale = fields.Boolean(compute="_compute_type_scale")
+    vehicle_id = fields.Many2one('vehicle.vehicle', string='Vehicle')
+    state = fields.Selection(selection_add=[('classified', 'Classified')])
```

Comment:
We've thought about this feature and seems too dangerous, so we'd prefer this approach:
- keep the current picking states.
- create a new `Many2one` field that represents "classification picking"
- when classified, fill the new above field, and cancel the current picking

With these changes, we'll be able to make a custom view for scale weights (those pickings that belongs to picking type with scale flag), and show if are pending of classification, or already classified, and ignore picking state. Finally, these pickings won't affect stock.